src/main/resources/jenkins/plugins/slack/user/SlackUserProperty/config.jelly

```diff
@@ -0,0 +1,9 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <f:entry title="Slack User Id">
+        <f:textbox field="userId"/>
+    </f:entry>
+    <f:entry title="Disable Notifications">
+        <f:checkbox field="disableNotifications"/>
```

Comment:
Could use a help message

Reply:
Agree:
![image](https://user-images.githubusercontent.com/21194782/62272179-62c5b780-b432-11e9-8c82-af9b27c0af58.png)

Also not sure if the text above needs to have slack in the user id?
it's in the slack section, could it just be "User Id" ?

Reply:
Yes, good suggestions.

Reply:
Here's what the config looks like with the latest changes:
![image](https://user-images.githubusercontent.com/551572/62279776-96d3b500-b3ff-11e9-8a3d-86e841f3f39c.png)

Reply:
@casz I don't see the benefit of adding an `f:section` around both the userId and disableNotfications entries:

![image](https://user-images.githubusercontent.com/551572/62313014-79750a00-b444-11e9-8a9b-dfd08565a1aa.png)

Reply:
> Here's what the config looks like with the latest changes:
> ![image](https://user-images.githubusercontent.com/551572/62279776-96d3b500-b3ff-11e9-8a3d-86e841f3f39c.png)

If it looks like that, just remove the section and have the setting say `Disable Slack notifications`